cmd/mount.go

```diff
@@ -399,6 +400,17 @@ var mountCmd = &cobra.Command{
 				Umask:       022,
 			}
 
+			if !daemon.WasReborn() {
+				// redirect libfuse stderr to a temp file
+				f, err := ioutil.TempFile("", "libfuse_stderr_")
```

Comment:
Instead of using tempFile can we convert this logic to use unix pipe instead. Be it success or failure in either case child writes down the message in pipe and parents reads from there. As soon as message is received parent can exit. If message was about failure parent can dump it on the screen as well.

Reply:
go-daemon uses forkExec to fork and exec a new child process, so it's impossible to use PIPE for communicating since they do not share opened file descriptor. 
We can use named PIPE or socket for communication, but I think that's no big difference with using a temp file.